modules/widgets/social-media-icons.php

```diff
@@ -72,6 +73,7 @@ public function __construct() {
 			'googleplus' => array( 'Google+', 'https://plus.google.com/u/0/%s/' ),
 			'flickr'     => array( 'Flickr', 'https://www.flickr.com/photos/%s/' ),
 			'wordpress'  => array( 'WordPress.org', 'https://profiles.wordpress.org/%s/' ),
+			'tumblr'     => array( 'Tumblr', 'https://%s.tumblr.com')
```

Comment:
You seem to be missing a space before the closing bracket here. I would also recommend adding a comma at the end of the line; it's best if each line is an array declaration ends in a comma.